### PR TITLE
Add Contributing Guide #531

### PR DESCRIPTION
**Pull request by salilbc**
# What Changed
Added Contributing.md file

# Why
Issue #531 

Todo:

- [ ] Add tests
- [ ] Add docs

<details>
  <summary>📦 Published PR as canary version: <code>2.5.5-canary.547.10454.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @design-systems/babel-plugin-include-styles@2.5.5-canary.547.10454.0
  npm install @design-systems/babel-plugin-replace-styles@2.5.5-canary.547.10454.0
  npm install @design-systems/cli-utils@2.5.5-canary.547.10454.0
  npm install @design-systems/cli@2.5.5-canary.547.10454.0
  npm install @design-systems/core@2.5.5-canary.547.10454.0
  npm install @design-systems/create@2.5.5-canary.547.10454.0
  npm install @design-systems/docs@2.5.5-canary.547.10454.0
  npm install @design-systems/eslint-config@2.5.5-canary.547.10454.0
  npm install @design-systems/load-config@2.5.5-canary.547.10454.0
  npm install @design-systems/plugin@2.5.5-canary.547.10454.0
  npm install @design-systems/stylelint-config@2.5.5-canary.547.10454.0
  npm install @design-systems/utils@2.5.5-canary.547.10454.0
  npm install @design-systems/build@2.5.5-canary.547.10454.0
  npm install @design-systems/bundle@2.5.5-canary.547.10454.0
  npm install @design-systems/clean@2.5.5-canary.547.10454.0
  npm install @design-systems/create-command@2.5.5-canary.547.10454.0
  npm install @design-systems/dev@2.5.5-canary.547.10454.0
  npm install @design-systems/lint@2.5.5-canary.547.10454.0
  npm install @design-systems/playroom@2.5.5-canary.547.10454.0
  npm install @design-systems/proof@2.5.5-canary.547.10454.0
  npm install @design-systems/size@2.5.5-canary.547.10454.0
  npm install @design-systems/storybook@2.5.5-canary.547.10454.0
  npm install @design-systems/test@2.5.5-canary.547.10454.0
  npm install @design-systems/update@2.5.5-canary.547.10454.0
  # or 
  yarn add @design-systems/babel-plugin-include-styles@2.5.5-canary.547.10454.0
  yarn add @design-systems/babel-plugin-replace-styles@2.5.5-canary.547.10454.0
  yarn add @design-systems/cli-utils@2.5.5-canary.547.10454.0
  yarn add @design-systems/cli@2.5.5-canary.547.10454.0
  yarn add @design-systems/core@2.5.5-canary.547.10454.0
  yarn add @design-systems/create@2.5.5-canary.547.10454.0
  yarn add @design-systems/docs@2.5.5-canary.547.10454.0
  yarn add @design-systems/eslint-config@2.5.5-canary.547.10454.0
  yarn add @design-systems/load-config@2.5.5-canary.547.10454.0
  yarn add @design-systems/plugin@2.5.5-canary.547.10454.0
  yarn add @design-systems/stylelint-config@2.5.5-canary.547.10454.0
  yarn add @design-systems/utils@2.5.5-canary.547.10454.0
  yarn add @design-systems/build@2.5.5-canary.547.10454.0
  yarn add @design-systems/bundle@2.5.5-canary.547.10454.0
  yarn add @design-systems/clean@2.5.5-canary.547.10454.0
  yarn add @design-systems/create-command@2.5.5-canary.547.10454.0
  yarn add @design-systems/dev@2.5.5-canary.547.10454.0
  yarn add @design-systems/lint@2.5.5-canary.547.10454.0
  yarn add @design-systems/playroom@2.5.5-canary.547.10454.0
  yarn add @design-systems/proof@2.5.5-canary.547.10454.0
  yarn add @design-systems/size@2.5.5-canary.547.10454.0
  yarn add @design-systems/storybook@2.5.5-canary.547.10454.0
  yarn add @design-systems/test@2.5.5-canary.547.10454.0
  yarn add @design-systems/update@2.5.5-canary.547.10454.0
  ```
</details>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Contributing
+
+We do not allow contributors to claim issues. If you find something interesting you can contribute to the repo, feel free to raise a PR
+
+1. Fork the repo
+2. Install dependencies locally `yarn`
+3. Make sure it builds `yarn start`
+4. Make your changes locally
+5. Ensure the code coverage is the same or higher than before your changes.
+6. Create a PR to the `master` branch
```
Comment (reubenae) [inline CONTRIBUTING.md:10]: Numbered lists in markdown should all begin with "1." the post processor will auto increment the list when displayed. You can verify this by checking how it renders in your branch https://github.com/salilbc/design-systems-cli/blob/master/CONTRIBUTING.md

Comment (reubenae) [reply CONTRIBUTING.md:10]: The reason for doing this is, in the future, if you want to add a new step at point 3, then you don't need to re-number all the following points as github will do it for you.

Comment (salilbc) [reply CONTRIBUTING.md]: Made the change

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in [`v2.14.0`](https://github.com/intuit/design-systems-cli/releases/tag/v2.14.0) :rocket:

**Pull request merged**